puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/client/1.21.groovy

```diff
@@ -2,8 +2,7 @@ pipeline {
     agent none
 
     environment {
-        version = "1.21"
-        branch = "rpm/${version}"
+        foreman_foreman_version = "1.21"
```

Comment:
why the double foreman?

Reply:
https://www.youtube.com/watch?v=hft1oYm_G2c